qiskit/chemistry/algorithms/minimum_eigen_solvers/vqe_adapt.py

```diff
@@ -49,7 +49,9 @@ def __init__(self, operator: LegacyBaseOperator,
                  initial_point: Optional[np.ndarray] = None,
                  excitation_pool: Optional[List[WeightedPauliOperator]] = None,
                  threshold: float = 1e-5,
-                 delta: float = 1, max_evals_grouped: int = 1,
+                 delta: float = 1,
+                 max_iterations: int = None,
```

Comment:
```suggestion
                 max_iterations: Optional[int] = None,
```
Since None is allowed it need to be Optional